openmdao/jacobians/jacobian.py

```diff
@@ -352,6 +349,8 @@ def _compute_sparsity(self, ordered_of_info, ordered_wrt_info, num_full_jacs, to
                     else:
                         J[roffset:rend, coffset:cend] = summ[key]
 
+        J /= num_full_jacs
+
```

Comment:
I believe J *= (1.0/num_full_jacs) is a lot faster.

Reply:
done